### ISSUE TITLE
Sort plots and images numerically

### ISSUE THREAD

**Issue by aurelien-m** [allegroai/clearml-server#156]
Is it possible to sort the images and the plots in the numerical order instead of the alphabetical order? Here's what it looks like (I've hidden the plots and images):

![Screenshot from 2022-08-22 12-18-43](https://user-images.githubusercontent.com/29719028/185909175-757fd779-02ca-4fd2-b5d7-3f4ed8f13f8d.png)
![Screenshot from 2022-08-22 12-19-03](https://user-images.githubusercontent.com/29719028/185909179-2749dcae-87c0-4349-9023-73b121f033f9.png)

It's especially for the plots where it's a bit annoying. Thanks! :smile:

**Comment by jkhenning** [comment]
Hi @aurelien-m , do you mean by the iteration number? As the title/series is a string (and even in your case, always starts with letters, how can we sort it numerically?

**Comment by aurelien-m** [comment]
Hey, I mean having `something 1`, `something 2`, `something 3`, ... and not `something 1`, `something 10`, `something 11`, ... `something 2`, `something 20`, `something 21` ... for example.

I have a bunch of graphs, but I'm not seeing them in the correct order. It feels a little strange! Here are my plots for example:

![image](https://user-images.githubusercontent.com/29719028/185970722-0c8c08e9-365e-453d-bb06-0dc75d07bc59.png)

So sorting it by iteration could help, it that's possible

**Comment by jkhenning** [comment]
Well, that would require actually parsing the string and understanding what to sort on, which is too general in the UI, I think.

Are you talking about sorting of the titles list, or of the plots themselves? I think the plots themselves are already sorted by iteration

**Comment by aurelien-m** [comment]
My plots are not sorted by iteration, you can see them go from 1 to 10 then it continues with 11, 12, .. 2, 20, 21, ... I tried adding an underscore in the title but it didn't change anything. I can try to look how it's sorted in the code, but I'm not sure if this is intended or not

**Comment by jkhenning** [comment]
This is intended since this is how a lexicographic sort would behave (these are strings, after all).

> My plots are not sorted by iteration, you can see them go from 1 to 10 then it continues with 11, 12, ..

That's in the "Find plots" list, as you shared (which is searchable), but is it also in the UI page that simply shows all plots for the task? This page by design shows the last iteration, than the one before, and so on.

**Comment by aurelien-m** [comment]
Yes it's also in the UI, my plots appear in the same order as the search "Find plots" list. You can see it in the images below, it goes in this order: 0, 1, 10, 11, 12, ... 18, 19, 2, 20, 21, ... and it doesn't match the iterations

![Screenshot from 2022-08-23 14-07-44](https://user-images.githubusercontent.com/29719028/186154313-2e547a3c-9287-4ca9-8ac2-c634b6d7382d.png)

![Screenshot from 2022-08-23 14-07-52](https://user-images.githubusercontent.com/29719028/186154321-9a0a13f0-7f1a-4810-af4e-34d24d64f32c.png)

**Comment by jkhenning** [comment]
Hi @aurelien-m ,

Sorry for the late response. I see the order is only for plots in the same iteration...?
